From c37450be0ee4ffe8615291e6a9f56aa4d272589d Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Date: Wed, 10 Apr 2019 14:05:38 +0530
Subject: [PATCH 0620/1852] ASoC: xlnx: revert HDMI audio mclk mutliplier

commit 149b7f3541155ab7e414a9989968c77df4364ffa from
https://github.com/Xilinx/linux-xlnx.git

Mutliplier value of 768 used in latest ACR IP is not providing proper
audio in some test cases. The old value of 512 is tested to be appropriate
with all tests. So, revert back to 512. ACR IP was aleady reverted back
to 512.

Signed-off-by: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xlnx_pl_snd_card.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/xilinx/xlnx_pl_snd_card.c b/sound/soc/xilinx/xlnx_pl_snd_card.c
index ab422771c53f..ab54827a677c 100644
--- a/sound/soc/xilinx/xlnx_pl_snd_card.c
+++ b/sound/soc/xilinx/xlnx_pl_snd_card.c
@@ -90,7 +90,7 @@ static int xlnx_hdmi_card_hw_params(struct snd_pcm_substream *substream,
 	case 96000:
 	case 176400:
 	case 192000:
-		prv->mclk_ratio = 768;
+		prv->mclk_ratio = 512;
 		break;
 	default:
 		return -EINVAL;
-- 
2.31.1

